alpakka/src/main/scala/org/scanamo/ops/retrypolicy/RetryPolicy.scala

```diff
@@ -39,39 +25,30 @@ sealed abstract class RetryPolicy extends Product with Serializable { self =>
   }
 
   final def update: RetryPolicy = self match {
-    case policy @ Constant(_, numberOfRetries)  => policy.copy(numberOfRetries = numberOfRetries - 1)(policy.scheduler)
-    case policy @ Linear(_, numberOfRetries, _) => policy.copy(numberOfRetries = numberOfRetries - 1)(policy.scheduler)
-    case policy @ Exponential(_, numberOfRetries, _) =>
-      policy.copy(numberOfRetries = numberOfRetries - 1)(policy.scheduler)
+    case policy: Constant            => policy
```

Comment:
If we are not changing anything in the policy then these cases will automatically cover by the last case (by default)

Reply:
Therefore, no need to add retry policy like Constant, Linear and Exponential if it's sound okay to you

Reply:
Yes 👍